### PR TITLE
Feature6/views

### PR DESCRIPTION
**Pull request by techEmprez**
### **The following was implemented in this PR**

- Implement the design from the sneak peek wireframes.
    - Styling is nice to have but focus on making sure everything functions first.     

- Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Choice, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has `node_modules` directory included in the project structure. Kindly ensure to exclude this directory from uploading to the repository by including the directory in the `.gitignore` file.

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/users/_user.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="sections">
+    <% unless user.photo.nil? %>
+        <%= image_tag (user.photo), height: '115', width: '115' %>
+    <% end %>
+    <div class="users">
+        <h1><%= link_to (user.name), user%></h1>
+        <span>Number of posts: <%= user.posts_counter %></span>    
+    </div>
+</div>
```
Comment (omarsalem7) [inline app/views/users/_user.html.erb:9]: - [ ] Great work for using partial view :100: :+1:

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,26 @@
...
+
+# Posts
+first_post = Post.create(author_id: first_user.id, title: 'Post1', text: 'This is my first post')
+second_post = Post.create(author_id: first_user.id, title: 'Post2', text: 'This is my second post')
+third_post = Post.create(author_id: first_user.id, title: 'Post3', text: 'This is my third post')
+fourth_post = Post.create(author_id: first_user.id, title: 'Post4', text: 'This is my fourth post')
+fifth_post = Post.create(author_id: second_user.id, title: 'Post5', text: 'This is my fifth post')
+
+# Comments
+Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.')
```
Comment (omarsalem7) [inline db/seeds.rb:21]: - [ ] Thanks for populating your database with data using seeds :100: :1st_place_medal:

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @techEmprez 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
- [ ] Kindly fix the MIT license link and add your own license.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/_user.html.erb (hunk 1 above)
Comment (techEmprez) [reply app/views/users/_user.html.erb:9]: Changes implemented on the READMe

File: db/seeds.rb (hunk 2 above)
Comment (techEmprez) [reply db/seeds.rb:21]: Thank you for you clear feedback

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @techEmprez       :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**